app/services/exports/export_service.rb

```diff
@@ -0,0 +1,51 @@
+module Exports
+  class ExportService
+    include CollectionTimeHelper
+
+    def initialize(storage_service, logger = Rails.logger)
+      @storage_service = storage_service
+      @logger = logger
+    end
+
+    def export_xml(full_update: false, collection_year: nil)
```

Comment:
Would it be worth adjusting this to allow us to trigger e.g. just a users export? (Maybe pass in a "collection" rather than "collection_year" and only run the relevant bits if provided?)

Reply:
Yes, I think so!